app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -114,12 +110,10 @@ You will also need a [visa or immigration status](/non-uk-teachers/visas-for-non
 
 ## Bursaries and scholarships for non-UK trainee languages and physics teachers
 
-$funding-holding-message$
-
 There are 2 types of financial support you can get if you’re interested in training to teach languages or physics:
 
-* bursaries for languages and physics
-* scholarships for French, German, Spanish and physics
+* bursaries worth up to $bursaries_postgraduate_nonuk$ for languages and physics
```

Comment:
Amount is displaying incorrectly due to incorrect value in `/values/bursaries` - no change needed here but just flagging for retesting